package.json

```diff
@@ -2,46 +2,77 @@
   "name": "@warp-ds/icons",
   "type": "module",
   "version": "0.0.1",
+  "repository": "git@github.com:warp-ds/icons.git",
   "description": "WARP's icon set",
   "main": "index.js",
   "files": [
-    "dist"
+    "dist",
+    "vue",
+    "react",
+    "elements"
   ],
   "exports": {
-    ".": "./index.js"
+    ".": "./index.js",
+    "./react": "./react/index.js",
+    "./vue": "./vue/index.js",
+    "./elements": "./elements/index.js",
+    "./elements/*": "./elements/*.js",
+    "./package.json": "./package.json"
   },
   "scripts": {
     "import": "node ./scripts/import.js",
     "build": "node ./scripts/build.js",
-    "render": "node preview/render.js",
     "dev": "vite",
     "vite:build": "vite build",
-    "preview": "pnpm run build && vite build && vite preview"
+    "preview": "pnpm run build && vite build && vite preview",
+    "build:release": "rimraf dist && pnpm run build && pnpm build:vue && pnpm build:react && pnpm build:elements",
+    "build:react": "rimraf react && mkdir react && node ./scripts/output/react.js && node ./react.eik.js",
+    "build:vue": "rimraf vue && mkdir vue && node ./scripts/output/vue.js && node ./vue.eik.js",
+    "build:elements": "rimraf elements && mkdir elements && node ./scripts/output/elements.js && node ./elements.eik.js"
   },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@eik/cli": "^2.0.22",
+    "@eik/esbuild-plugin": "^1.1.11",
     "@sindresorhus/slugify": "^2.1.0",
     "@warp-ds/uno": "^1.0.0",
-    "camelcase": "^7.0.0",
+    "camelcase": "^8.0.0",
     "chalk": "^5.0.1",
     "esbuild": "^0.14.42",
-    "fs-extra": "^10.1.0",
-    "glob": "^8.0.3",
-    "happy-dom": "^6.0.4",
+    "fs-extra": "^11.1.1",
+    "glob": "^10.0.0",
+    "happy-dom": "^10.0.0",
     "js-yaml": "^4.1.0",
-    "jsdom": "^16.5.3",
     "nanoid": "^4.0.0",
     "node-fetch": "^3.2.9",
-    "nunjucks": "^3.2.2",
-    "ora": "^6.1.2",
+    "ora": "^7.0.1",
     "prompts": "^2.4.0",
-    "rimraf": "^3.0.2",
+    "rimraf": "^5.0.1",
     "svgo": "^3.0.0",
     "unocss": "^0.55.2",
     "vite": "^4.4.9",
     "vite-plugin-nunjucks": "^0.1.10"
   },
   "publishConfig": {
     "access": "public"
+  },
+  "eik": {
+    "server": "https://assets.finn.no",
+    "type": "package",
+    "files": {
+      "react": "./dist/react",
+      "vue": "./dist/vue",
+      "elements": "./dist/elements"
+    },
+    "import-map": [
+      "https://assets.finn.no/map/react/v2",
+      "https://assets.finn.no/map/vue/v1",
+      "https://assets.finn.no/map/custom-elements/v2"
+    ]
```

Comment:
Ah! That's what was missing for the imports to be mapped to eik urls in the built files! 👏